### DISCUSSION TITLE
Cilium Cluster Mesh & Talos

### DISCUSSION THREAD

**Discussion by allright** [siderolabs/talos#7948] [General]
Has anyone tested cilium cluster mesh on Talos?
https://cilium.io/use-cases/cluster-mesh/

Does it work well?
The goal is to create 3 connected clusters in different locations.

P.S.
@smira @Unix4ever 
  Hello from Qik :)

**Comment by smira** [comment]
Hi Andrey! Nice to meet you once again after so many years!

I do not have any information about Cilium Cluster Mesh. I guess it should work, there's nothing in Talos that might prevent it from working.

**Comment by allright** [comment]
🙏, will test :)

**Comment by teaglebuilt** [comment]
i am testing this now. any updates from others?
